lib/archethic/utils/regression/benchmarks/helpers/internal_faucet.ex

```diff
@@ -0,0 +1,92 @@
+defmodule ArchEthic.Utils.Regression.Benchmarks.Helpers.InternalFaucet do
+  @moduledoc """
+  Executes  UCO withrawl internally to the Tx-blockchain.
+  """
+
+  alias ArchEthicWeb.TransactionSubscriber
+  alias ArchEthic.Crypto
+
+  alias ArchEthic.TransactionChain.{
+    Transaction,
+    TransactionData,
+    TransactionData.Ledger,
+    TransactionData.UCOLedger
+  }
+
+  alias ArchEthicWeb.TransactionSubscriber
+
+  @pool_seed Application.compile_env(:archethic, [ArchEthicWeb.FaucetController, :seed])
+
+  def main(recipient_address) do
```

Comment:
I think you should pass a list recipients , so in a transaction you can fund multiple clients which in their turn will emit x amount of transactions in parallel.

Reply:
I want information regarding building a transaction for multiple recipients.
this_way?
```
send_transaction(
      @faucet_seed,
      :transfer,
      %TransactionData{
        ledger: %Ledger{
          uco: %UCOLedger{
            transfers: [
              %UCOTransfer{
                to: [address_in_binary,address_in_binary],
                amount: amount * 100_000_000
              }
            ]
          }
        }
      },
      host,
      port,
      :ed25519
    )
```

or this_way ? 
```send_transaction(
      @faucet_seed,
      :transfer,
      %TransactionData{
        ledger: %Ledger{
          uco: %UCOLedger{
            transfers: [
              %UCOTransfer{
                to: address_in_binary,
                amount: amount * 100_000_000
              },
               %UCOTransfer{
                to: address_in_binary,
                amount: amount * 100_000_000
              },%UCOTransfer{
                to: address_in_binary,
                amount: amount * 100_000_000
              }
            ]
          }
        }
      },
      host,
      port,
      :ed25519
    )```